python/ml4ir/base/features/feature_config.py

```diff
@@ -335,6 +335,25 @@ def get_feature_by_node_name(self, name: str):
 
         raise KeyError("No feature with node name {} in FeatureConfig".format(name))
 
+    def get_old_ranking_score(self, name: str):
+        """
+        Getter method for the old ranking score feature in FeatureConfig object
+
+        Parameters
+        ----------
+        name : str
+            Name of the feature to fetch
+
+        Returns
+        -------
+        dict
+            Feature config dictionary for the name of the feature passed
+        """
+        try:
+            return self.get_feature(name)
+        except KeyError:
+            return None
+
```

Comment:
@mohazahran I don't see the point of this function. You should remove it.

Reply:
done